symengine/polys/uintpoly_flint.h

```diff
@@ -74,6 +74,17 @@ class UFlintPoly : public BaseType<Container, Poly>
         return make_rcp<const Poly>(var, std::move(f));
     }
 
+    template <typename FromPoly>
+    static RCP<const Poly> from_poly(const FromPoly &p)
```

Comment:
Can you enable this only if `FromPoly` is a univariate polynomial? Otherwise, it looks good.